src/CitMovie.Api/Controller/FollowController.cs

```diff
@@ -33,7 +34,7 @@ public async Task<IActionResult> GetFollowings(int userId, int page = 0, int pag
 
 
     [HttpPost]
-    public async Task<ActionResult<FollowDto>> CreateFollow(int userId, [FromBody] CreateFollowDto createFollowDto)
+    public async Task<ActionResult<FollowResult>> CreateFollow(int userId, [FromBody] CreateFollowResult createFollowDto)
```

Comment:
The type of the body should be `FollowCreateRequest` instead of `CreateFollowResult`